### PR TITLE
Add Tabs component

### PR DESCRIPTION
**Pull request by dgca**

### CODE REVIEW

File: packages/tackle-box/lib/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+type ListProps = {
+  children: ReactNode;
+};
+
+function List({ children }: ListProps) {
+  return (
+    <HStack justifyContent="center" borderBottomWidth={1} borderColor="divider">
+      {children}
```
Comment (cgjohn) [inline packages/tackle-box/lib/components/Tabs/Tabs.tsx:41]: If we want to make things more accessible, we should have the children here be a `ul` and then we can wrap the trigger elements in an `li`.

The `ul` can then also have the role of `tablist` and each `li` can have a role of `tab`

File: packages/tackle-box/lib/components/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+type ContentProps = {
+  children: ReactNode;
+  value: string;
+};
+
+function Content({ children, value }: ContentProps) {
+  const context = useTabsContext();
+
+  return context.activeTab === value ? <>{children}</> : null;
```
Comment (cgjohn) [inline packages/tackle-box/lib/components/Tabs/Tabs.tsx:89]: the content of a tab content area should have a role of `tabpanel`

File: packages/tackle-box/lib/components/Tabs/Tabs.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  },
+});
+
+function Trigger({ children, value }: TriggerProps) {
+  const context = useTabsContext();
+  const isActive = context.activeTab === value;
+
+  return (
+    <html.button
+      onClick={() => context.setActiveTab(value)}
```
Comment (cgjohn) [inline packages/tackle-box/lib/components/Tabs/Tabs.tsx:65]: if active we can add an `aria-selected` value here so it reads out as the active tab

**Review by cgjohn — APPROVED** [review]
only thoughts are around a11y

**Comment by dgca** [comment]
Talked on Slack and decided to fix the a11y stuff as a follow up after trying to make <Box /> polymorphic

**Pull request merged**